### ISSUE TITLE
V4 future

### ISSUE THREAD

**Issue by dmwyatt** [mdbootstrap/mdb-ui-kit#1141]
This isn't a criticism, just a question to help me choose a frontend framework: 

I see it's been 6 months since any substantial commits to the v4 project.  Am I missing where active development is happening or are there just not enough devs/time to push it forward?

**Comment by FezVrasta** [comment]
1. still waiting for a stable bootstrap 4 release to avoid to write useless code on stuff that will change
2. not enough contributors here, I'll do my best to release v4 once bs4 is ready

I hope it answers your questions

**Comment by FezVrasta** [comment]
Talking about this, any help to rebase our v4 branch to the latest v4 branch of Bootstrap would be awesome.

Also, the jekyll website on our v4 seems broken, I hate Ruby, I hate Jekyll, and in fact, it doesn't work anymore without any good reason... If someone is able to fix it please do it.

**Comment by FezVrasta** [comment]
Closing this in favor of https://github.com/FezVrasta/bootstrap-material-design/issues/1166
